integrations/aws/pyproject.toml

```diff
@@ -6,7 +6,8 @@ authors = ["Shalev Avhar <shalev@getport.io>", "Erik Zaadi <erik@getport.io>"]
 
 [tool.poetry.dependencies]
 python = "^3.11"
-port_ocean = {version = "^0.9.7", extras = ["cli"]}
+port_ocean = {extras = ["cli"], version = "^0.9.7"}
+
 python-dotenv = "^1.0.1"
 aioboto3 = "^12.4.0"
 boto3-stubs = {version = "1.34.76", extras = ["acm", "apigateway", "appconfig", "athena", "cloudcontrol", "cloudformation", "cloudwatch", "dynamodb", "ec2", "ec2-instance-connect", "ecr", "ecs", "elasticache", "elb", "elbv2", "events", "iam", "lambda", "logs", "organizations", "rds", "route53", "s3", "sagemaker", "secretsmanager", "sns", "sqs", "ssm", "sts"]}
```

Comment:
You forgot to bump the integration version itself

[tool.poetry]
name = "aws"
version = "new_version"